### PR TITLE
Add basic logging for the work generator

### PR DESCRIPTION
**Pull request by bjcoleman**
Logging looks like:

```
➜  mirrulations git:(work_gen_logging) ✗ docker-compose logs -ft work_generator
Attaching to mirrulations_work_generator_1
work_generator_1  | 2023-03-02T20:11:00.233280211Z Begin generate docket jobs
work_generator_1  | 2023-03-02T20:11:10.330946300Z Added dockets: 1
work_generator_1  | 2023-03-02T20:11:10.331227175Z End generate docket jobs
work_generator_1  | 2023-03-02T20:11:10.331238966Z Begin generate document jobs
work_generator_1  | 2023-03-02T20:11:16.068664761Z Added documents: 250
work_generator_1  | 2023-03-02T20:11:21.837571763Z Added documents: 250
work_generator_1  | 2023-03-02T20:11:27.557239252Z Added documents: 250
```

### CODE REVIEW

File: mirrulations-work-generator/src/mirrgen/work_generator.py [hunk 1]
```diff
@@ -55,9 +55,15 @@ def generate_work():
...
         # from all jobs in the job queue
+        print('Begin generate docket jobs')
         generator.download('dockets')
+        print('End generate docket jobs')
+        print('Begin generate document jobs')
         generator.download('documents')
+        print('End generate document jobs')
+        print('Begin generate comment/attachment jobs')
         generator.download('comments')
+        print('End generate commend/attachment jobs')
```
Comment (justinszaro) [inline mirrulations-work-generator/src/mirrgen/work_generator.py:66]: Quick spelling error here. Can you change this to comment?

Comment (bjcoleman) [reply mirrulations-work-generator/src/mirrgen/work_generator.py]: done

**Comment by justinszaro** [comment]
Thanks. If you are ready, you can merge.

**Comment by bjcoleman** [comment]
Yes, this is ready

**Pull request merged**